README-V3-UPGRADE.md

```diff
@@ -21,6 +33,14 @@ The packages in `org.spdx.licenseTemplates` are now in the `java-spdx-core` repo
 
 A new class `LicenseTextHelper` was added and the method `isLicenseTextEquivalent(String, String)` along with many supporting methods were moved to `LicenseTextHelper` from `org.spdx.utility.compare.LicenseCompareHelper`.
 
+## Other moved classes
+
+- `org.spdx.library.model.license.ListedLicenses` moved to `org.spdx.library.ListedLicenses`
+
+## Removed methods
+
+Some methods that exist in the v2.3 model classes (e.g. `org.spdx.library.model.license.SpdxListedLicense.getLicenseTextHtml()`, `org.spdx.library.model.license.SpdxListedLicense.getCrossRef()`, `org.spdx.library.model.license.ListedLicenseException.getExceptionTextHtml()`) have been removed and do not have an equivalent in the v3.0 model classes.
```

Comment:
@goneall please confirm that this is true. I couldn't find where these methods went, but I'm not 100% sure it doesn't in fact exist somewhere in one of the other new libraries.

Reply:
This is true - the SPDX 3 specification does not define these three fields.

A bit of background - the 3 fields were added to the SPDX 2 model to enable the license list publisher to use the standard model for publishing.  The 3 fields never made it into the spec, however.  In hindsight, we probably should have created a subclass of listed license to store the additional fields - but I don't want to implement something that may break compatibility.

@zvr @swinslow  - do you see any value of adding the information used to generate the license website to the actual listed license model?  If so, we can propose adding them back to the SPDX 3 model.